src/current/v25.2/create-policy.md

```diff
@@ -0,0 +1,113 @@
+|-------------------------------------|-----------------------------------------------------|--------------------------------------------------|---------------------------------------------------|-------------------------------------------------------|-----------------------------------------------|
+| `SELECT`                            | ✓                                                   | —                                                | —                                                 | —                                                     | —                                             |
+| `SELECT ... FOR UPDATE / FOR SHARE` | ✓                                                   | —                                                | ✓                                                 | —                                                     | —                                             |
+| `INSERT / MERGE ... THEN INSERT`    | —                                                   | ✓                                                | —                                                 | —                                                     | —                                             |
+| `INSERT ... RETURNING`              | ✓ ‡                                                 | ✓                                                | —                                                 | —                                                     | —                                             |
+| `UPDATE / MERGE ... THEN UPDATE`    | ✓ ‡                                                 | —                                                | ✓                                                 | ✓                                                     | —                                             |
+| `DELETE`                            | ✓ ‡                                                 | —                                                | —                                                 | —                                                     | ✓                                             |
+| `INSERT ... ON CONFLICT DO UPDATE`  | ✓                                                   | —                                                | ✓                                                 | ✓                                                     | —                                             |
+
+‡ The `SELECT` check is only evaluated when the statement actually needs to read from the relation, e.g., in a `WHERE`, `SET`, or `RETURNING` clause that references table columns.
+
+CockroachDB's behavior differs from that described in the table above in the following ways:
+
+- `SELECT` evaluation: CockroachDB always evaluates `SELECT` (`USING`) policies for `INSERT` and `UPDATE`, even when the statement doesn't reference table columns. This differs from PostgreSQL; this difference may be resolved in an upcoming release.
```

Comment:
It applies to DELETE as well. Not sure if it matters, but it will be included in 25.3 and likely backported to 25.2.x.

Reply:
I've updated it to say

> - `SELECT` evaluation: CockroachDB always evaluates `SELECT` (`USING`) policies for `INSERT`, `UPDATE`, and `DELETE`, even when the statement doesn't reference table columns.

and removed the PG/upcoming release stuff - removed the PG ref per your previous comment; and our policy is to avoid making future comments in the docs but i mistakenly did so there